tests/Utilities/MakeWithRandomValues.hpp

```diff
@@ -70,6 +70,53 @@ struct FillWithRandomValuesImpl<Variables<tmpl::list<Tags...>>,
 /// \endcond
 }  // namespace TestHelpers_detail
 
+/// \ingroup TestingFrameworkGroup
+/// \brief A uniform distribution functor for which the behavior can be easily
+/// customized
+///
+/// \details The two constructors take either a low and high bound arguments or
+/// a similar 2-component array , which can be passed in in any type from which
+/// the generated type can be constructed. The default generation function
+/// operates as would be expected for a double
+template <typename T>
+struct uniform_custom_distribution {
```

Comment:
How is this better than fill_with_random_values?

Reply:
it's a level lower - fill_with_random_values takes a generator and a distribution as arguments, so you need a distribution to provide. And the std:: distributions are unsatisfactory for generating complex numbers as well as other types